### PR TITLE
test MBW in pPb

### PR DESCRIPTION
**Pull request by hmurakam**

### CODE REVIEW

File: PWGGA/GammaConv/AliAnalysisTaskMaterialHistos.cxx [hunk 1]
```diff
@@ -320,6 +330,7 @@ void AliAnalysisTaskMaterialHistos::UserCreateOutputObjects()
     fDeDxMapList            = new TList*[fnCuts];
   }
   hNEvents                  = new TH1F*[fnCuts];
+  hBCNumber                 = new TH1F*[fnCuts];
```
Comment (nschmidtALICE) [inline PWGGA/GammaConv/AliAnalysisTaskMaterialHistos.cxx]: i recommend to put the if(fDoSelectBCNumber) condition on the creation of this histogram.

File: PWGGA/GammaConv/AliAnalysisTaskMaterialHistos.cxx [hunk 2]
```diff
@@ -458,6 +469,8 @@ void AliAnalysisTaskMaterialHistos::UserCreateOutputObjects()
     hNEvents[iCut]->GetXaxis()->SetBinLabel(14,"Pileup V0M-TPCout Tracks");
     fESDList[iCut]->Add(hNEvents[iCut]);
 
+    hBCNumber[iCut]              = new TH1F("BCNumber","BCNumber",3564,-0.5,3563.5);
+    fESDList[iCut]->Add(hBCNumber[iCut]);
```
Comment (nschmidtALICE) [inline PWGGA/GammaConv/AliAnalysisTaskMaterialHistos.cxx]: same comment as above.

File: PWGGA/GammaConv/AliAnalysisTaskMaterialHistos.cxx (hunk 1 above)
Comment (hmurakam) [reply PWGGA/GammaConv/AliAnalysisTaskMaterialHistos.cxx]: thanks you for the advice !

File: PWGGA/GammaConv/AliAnalysisTaskMaterialHistos.cxx [hunk 3]
```diff
@@ -457,8 +470,10 @@ void AliAnalysisTaskMaterialHistos::UserCreateOutputObjects()
     hNEvents[iCut]->GetXaxis()->SetBinLabel(13,"Out-of-Bunch pileup Past-Future");
     hNEvents[iCut]->GetXaxis()->SetBinLabel(14,"Pileup V0M-TPCout Tracks");
     fESDList[iCut]->Add(hNEvents[iCut]);
-
-
+    if(fDoSelectBCNumber)
```
Comment (nschmidtALICE) [inline PWGGA/GammaConv/AliAnalysisTaskMaterialHistos.cxx]: you are missing a "{" bracket here

**Comment by nschmidtALICE** [comment]
+1

**Pull request merged**